src/chapters/sql-part1/joins.rst

```diff
@@ -0,0 +1,152 @@
+
+   .. figure:: figures/innerjoin.png
+      :alt: Venn diagram highlighting just the center where the two circles meet.
+
+   The venn diagram above shows the result set highlighted in blue.
+
+.. index:: ! left outer join
+
+Left Outer Join
+---------------
+
+Joining two tables on a **left outer join** gives us a result set will include all values in the left table and any matching records from the right table.
+
+.. admonition:: Example 
+
+   If we use a left inner join to combine ``JohnsonWedding`` and ``JohnsonVowRenewal`` in a query, we can see what all of the guests that were invited to the wedding and any guests who were also invited to the vow renewal.
+
+   .. sourcecode:: mysql
+
+      SELECT LastName, FirstName
+      FROM JohnsonWedding
+      LEFT OUTER JOIN JohnsonVowRenewal ON JohnsonWedding.GuestID == JohnsonVowRenewal.GuestID;
+
+   .. figure:: figures/leftouterjoin.png
+      :alt: Venn diagram highlighting the center and entirety of left circle.
+
+   The venn diagram above shows the result set highlighted in blue.
+
+.. index:: ! right outer join
+
+Right Outer Join
+----------------
+
+Joining two tables on a **right outer join** gives us a result set that includes all values in the right table and any matching records from the left table.
+
+.. admonition:: Example 
+
+   If we use a right inner join to combine ``JohnsonWedding`` and ``JohnsonVowRenewal`` in a query, we can see what all of the guests that were invited to the vow renewal and any guests who were also invited to the wedding.
+
+   .. sourcecode:: mysql
+
+      SELECT LastName, FirstName
+      FROM JohnsonWedding
+      RIGHT OUTER JOIN JohnsonVowRenewal ON JohnsonWedding.GuestID == JohnsonVowRenewal.GuestID;
+
+   .. figure:: figures/rightouterjoin.png 
+      :alt: Venn diagram highlighting the center and entirety of right circle.
+
+   The venn diagram above shows the result set highlighted in blue.  
+
+.. index:: ! full outer join
+
+Full Outer Join
+---------------
+
+Joining two tables on a **full outer join** gives us a result set that includes all records from both tables. 
+Full outer joins are important to SQL, but the syntax is not supported in MySQL.
+Instead, to achieve a full outer join, you have to work with a left outer join and a right outer join.
+To show what a full outer join looks like in other types of SQL, we have simulated some possible syntax below.
+
+.. admonition:: Example
+
+   Now that another event planner has joined Mary's company, to get all of the events run by the company in August, we can use a full outer join to combine ``MaryEvents`` and ``LeahEvents``.
+
+   .. sourcecode:: mysql
+
+      SELECT *
+      FROM MaryEvents
+      FULL OUTER JOIN LeahEvents ON MaryEvents.Month == LeahEvents.Month
+      WHERE MaryEvents.Month == 08;
+
+   .. figure:: figures/fullouterjoin.png
+      :alt: Venn diagram with the entirety of both circles highlighted.
```

Comment:
Since this syntax does not work in MySQL, can we have two code blocks here, one showing the FULL JOIN syntax and the other showing how to accomplish the same thing with LEFT and RIGHT?

Reply:
I added a note! Let me know your thoughts!